SourceryFramework/Sources/Parsing/Utils/InlineParser.swift

```diff
@@ -60,7 +60,22 @@ public enum TemplateAnnotationsParser {
             } else {
                 annotatedRanges[name] = [(range: range, indentation: indentation)]
             }
-            rangesToReplace.insert(range)
+            var rangeToBeRemoved = true
+            
+            for ignoreTemplate in forceParse {
+                
+                
+                if name.hasSuffix("." + ignoreTemplate) {
+                    rangeToBeRemoved = false
+                    break
+                }
+            }
+            
+            if rangeToBeRemoved || name.hasSuffix("." + "AutoCoding") {
+             //   print("removed named range \(name) and \(forceParse)")
```

Comment:
lets remove this commented out code

Reply:
I'm also not sure why AutoCoding is explicitly used here

Reply:
How to add changelog?

Reply:
> I'm also not sure why AutoCoding is explicitly used here
Purpose of using a default value is to allow all tests to pass.